lib/Cargo.toml

```diff
@@ -44,6 +44,7 @@ toml = "0.8.12"
 xshell = { version = "0.2.6", optional = true }
 uuid = { version = "1.8.0", features = ["v4"] }
 tini = "1.3.0"
+struct_iterable = "0.1.1"
```

Comment:
This crate is [not used very much](https://lib.rs/crates/struct_iterable/rev)...a bit of a red flag.